stencil.config.ts

```diff
@@ -99,7 +100,7 @@ export const create: () => Config = () => ({
       injectGlobalPaths: ["src/assets/styles/includes.scss"]
     }),
     postcss({
-      plugins: [tailwind(), autoprefixer()]
+      plugins: [tailwindcss(tailwindConfig), autoprefixer()]
```

Comment:
FYI, I installed a PR that will likely cause a merge conflict with the `tailwindcss` plugin.